.github/workflows/release_alpha.yml

```diff
@@ -0,0 +1,147 @@
+name: Build and push image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Docker tag'
+        required: true
+
+jobs:
+  build:
+    name: Run Gitian build
+    runs-on: self-hosted
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: dash
+    
+    - name: Checkout Gitian builder
+      uses: actions/checkout@v2
+      with:
+        repository: devrandom/gitian-builder
+        path: gitian-builder
+
+    - name: Checkout detached sigs
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/dash-detached-sigs
+        path: dash-detached-sigs
+
+    - name: Checkout gitian sigs
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/gitian.sigs
+        path: gitian.sigs
+
+    - name: Import GPG key
+      id: import_gpg
+      uses: crazy-max/ghaction-import-gpg@v4
+      with:
+        gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+        passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+    - name: GPG user IDs
+      run: |
+        echo "fingerprint: ${{ steps.import_gpg.outputs.fingerprint }}"
+        echo "keyid:       ${{ steps.import_gpg.outputs.keyid }}"
+        echo "name:        ${{ steps.import_gpg.outputs.name }}"
+        echo "email:       ${{ steps.import_gpg.outputs.email }}"
+
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: dash
+    
+    - name: Checkout tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: devrandom/gitian-builder
+        path: gitian-builder
+
+    - name: Checkout tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/dash-detached-sigs
+        path: dash-detached-sigs
+
+    - name: Checkout tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/gitian.sigs
+        path: gitian.sigs
+
+    # - name: Clone macOS SDK
+    #   run: |
+    #     mkdir gitian-builder/inputs
+    #     wget -q -O gitian-builder/inputs/MacOSX10.11.sdk.tar.gz https://bitcoincore.org/depends-sources/sdks/MacOSX10.11.sdk.tar.gz
+
+    - name: Import GPG key
+      run: echo ${{GPG_PRIVATE_KEY}} | gpg --import
+
+    - name: Docker meta
+      id: docker_meta
+      uses: docker/metadata-action@v3
+      with:
+        images: strophy/dashd
```

Comment:
this probably isn't okay?

Reply:
This will be the last thing I change before merging